2025-07-28 14:20:48,457 - telegram.ext.Application - INFO - Application started
2025-07-28 14:20:49,293 - root - INFO - User (7317705516) in private: 'hello'
2025-07-28 14:20:49,293 - root - INFO - Bot response: Hey there!
2025-07-28 14:21:10,882 - root - INFO - User (7317705516) in private: 'hello'
2025-07-28 14:21:10,882 - root - INFO - Bot response: Hey there!
2025-07-28 14:21:44,919 - root - INFO - User (7317705516) in private: 'how are you'
2025-07-28 14:21:44,919 - root - INFO - Bot response: I'm fine, thank you!
2025-07-28 14:21:52,557 - telegram.ext.Application - INFO - Application is stopping. This might take a moment.
2025-07-28 14:21:52,558 - telegram.ext.Application - INFO - Application.stop() complete
2025-07-28 15:59:13,970 - telegram.ext.Application - INFO - Application started
2025-07-28 15:59:15,097 - root - INFO - User (7317705516) in private: 'hello'
2025-07-28 15:59:15,097 - root - INFO - Bot response: Hey there!
2025-07-28 15:59:26,771 - root - INFO - User (7317705516) in private: 'hello'
2025-07-28 15:59:26,771 - root - INFO - Bot response: Hey there!
2025-07-28 15:59:35,064 - root - INFO - User (7317705516) in private: 'how are you'
2025-07-28 15:59:35,064 - root - INFO - Bot response: I'm fine, thank you!
2025-07-28 15:59:43,870 - telegram.ext.Application - INFO - Application is stopping. This might take a moment.
2025-07-28 15:59:43,870 - telegram.ext.Application - INFO - Application.stop() complete
2025-07-29 12:21:19,467 - telegram.ext.Application - INFO - Application started
2025-07-29 12:21:33,610 - root - INFO - User (7317705516) in private: 'Hello'
2025-07-29 12:21:33,610 - root - INFO - Bot response: Hey there!
2025-07-29 12:21:45,168 - root - INFO - User (7317705516) in private: 'How are you doing'
2025-07-29 12:21:45,168 - root - INFO - Bot response: I'm fine, thank you!
2025-07-29 12:21:56,214 - root - ERROR - Update Update(message=Message(channel_chat_created=False, chat=Chat(first_name='Onyi', id=7317705516, last_name='Kachi ü¶¥üêæ', type=<ChatType.PRIVATE>, username='GeorgeTero'), date=datetime.datetime(2025, 7, 29, 16, 21, 51, tzinfo=datetime.timezone.utc), delete_chat_photo=False, entities=(MessageEntity(length=5, offset=0, type=<MessageEntityType.BOT_COMMAND>),), from_user=User(first_name='Onyi', id=7317705516, is_bot=False, language_code='en', last_name='Kachi ü¶¥üêæ', username='GeorgeTero'), group_chat_created=False, message_id=134, supergroup_chat_created=False, text='/help'), update_id=678871846) caused error Timed out
2025-07-29 12:22:26,235 - telegram.ext.Application - INFO - Application is stopping. This might take a moment.
2025-07-29 12:22:26,235 - telegram.ext.Application - INFO - Application.stop() complete
2025-07-29 14:00:14,677 - telegram.ext.Application - INFO - Application started
2025-07-29 14:00:25,318 - root - INFO - User (7317705516) in private: 'hello'
2025-07-29 14:00:25,318 - root - INFO - Bot response: Hey there!
2025-07-29 14:00:35,148 - root - INFO - User (7317705516) in private: 'how are you doing'
2025-07-29 14:00:35,148 - root - INFO - Bot response: I'm fine, thank you!
2025-07-29 14:02:25,840 - telegram.ext.Application - INFO - Application is stopping. This might take a moment.
2025-07-29 14:02:25,840 - telegram.ext.Application - INFO - Application.stop() complete
